backend/organization/models/project.py

```diff
@@ -181,6 +182,15 @@ class Project(models.Model):
         on_delete=models.SET_NULL,
     )
 
+    @property
+    def ranking(self) -> int:
+        return ProjectRanking().calculate_ranking(
+            location=self.loc,
+            project_id=self.id,
+            total_comments=self.total_comments,
+            total_followers=self.total_followers,
+            total_likes=self.total_likes
+        )
```

Comment:
This is calculated when you call the method. I call project.ranking on the projects API. So it will be calculated on the fly when the page is loading.